cmd/tiller/release_server.go

```diff
@@ -525,7 +530,7 @@ func (s *releaseServer) UninstallRelease(c ctx.Context, req *services.UninstallR
 		return nil, errMissingRelease
 	}
 
-	rel, err := s.env.Releases.Get(req.Name)
+	rel, err := s.env.Releases.Get(req.Name, 0)
```

Comment:
also can't delete in memory. Should be 1 instead of 0. Maybe the defaulting to 1 just isn't hooked up. 
```
helm [(detached from brian/master)*]$ bin/helm install docs/examples/alpine/
Error: no available release name found
```